src/test/datascience/notebook/notebookEditorProvider.vscode.test.ts

```diff
@@ -138,7 +138,6 @@ suite('DataScience - VSCode Notebook (Editor Provider)', function () {
         await commandManager.executeCommand('vscode.open', Uri.file(emptyPyFile));
 
         await activeNotebookChanged.assertFired();
-        assert.isTrue(notebookClosed.fired, 'Unpinned notebook should have been closed when opening another file');
```

Comment:
Line 162 performs the test (this was never a correct test)

Reply:
New untitled files seem to be unpinned for me in VS Code Insiders. Did I misunderstand your comment?

Reply:
Not for me, untitled notebooks.
Either way it's ok, were testing the events, whether it's pinned or not for untitled didn't matter, free existing failed it's definitely not pinned.